helm-charts/helm3/strimzi-kafka-operator/templates/NOTES.txt

```diff
@@ -0,0 +1,5 @@
+Thank you for installing {{ .Chart.Name }}-{{ .Chart.Version }}
+
+To create a Kafka cluster refer to the following documentation.
+
+https://strimzi.io/docs/operators/latest/using.html#deploying-cluster-operator-helm-chart-str
```

Comment:
Those are instructions for installing the CO using helm. IDK when a user of the chart would see these notes, but the sentence "to create a Kafka cluster refer to the following documentation" suggests it would be _after_ they've installed the CO, therefore, would https://strimzi.io/docs/operators/0.18.0/using.html#kafka-cluster-str not be a better place to refer them to?

I think we should use `{{ .Chart.Version }}` rather than latest.